pages/linux/plasma-apply-wallpaperimage.md

```diff
@@ -0,0 +1,16 @@
+# plasma-apply-wallpaperimage
+
+> Switch Plasma wallpaper.
+> More information: <https://invent.kde.org/plasma/plasma-workspace>.
+
+- Apply a wallpaper:
+
+`plasma-apply-wallpaperimage {{/path/to/image}}`
+
+- Apply a wallpaper with a fill mode:
+
+`plasma-apply-wallpaperimage --fill-mode {{stretch|preserveAspectFit|preserveAspectCrop|tile|tileVertically|tileHorizontally|pad}} {{/path/to/image}}`
```

Comment:
```suggestion
`plasma-apply-wallpaperimage --fill-mode {{stretch|preserveAspectFit|preserveAspectCrop|tile|tileVertically|tileHorizontally|pad}} {{path/to/image}}`
```

Reply:
This is intentional. Absolute path is a must.